contrib/systemd/system/zfs-import-cache.service.in

```diff
@@ -4,10 +4,11 @@ DefaultDependencies=no
 Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
-ConditionPathExists=@sysconfdir@/zfs/zpool.cache
+ConditionPathExists=${ZFS_CACHE}
```

Comment:
Dang! That have been there for a very long time, and no one have noticed! Kind'a indicates how few that actually use the cache file!